src/ORM/Marshaller.php

```diff
@@ -439,6 +449,59 @@ protected function _loadBelongsToMany($assoc, $ids)
     }
 
     /**
+     * Call translation merge
+     *
+     * @param \Cake\Datasource\EntityInterface $original The original entity
+     * @param array $data key value list of languages with fields to be merged into the translate entity
+     * @param array $options list of options
+     * @return array
+     */
+    protected function _mergeTranslations($original, array $data, array $options = [])
+    {
+        $result = $this->_table->mergeTranslations($original, $data, $this, $options);
```

Comment:
Would it be better if `$this` was the first argument? Then the method would be 'currying' in the additional parameter.